pallets/msa/src/tests.rs

```diff
@@ -955,3 +982,101 @@ pub fn delegation_expired() {
 		);
 	})
 }
+
+#[test]
+fn signed_extension_revoke_msa_delegation_by_delegator() {
```

Comment:
These tests are long enough that Reviewer and Future Developer could use some comments to understand what's going on and what is being tested.

Reply:
Comments added to this test